lib/file-system.js

```diff
@@ -29,7 +38,7 @@ exports.mkdirp = function(path) {
       else resolve();
     });
   });
-}
+};
```

Comment:
could you undo this change (and similar ones below)? a different PR fixes linting issues so we should try and prevent merge conflicts here